api/BrowserCaptureMediaStreamTrack.json

```diff
@@ -0,0 +1,103 @@
+{
+  "api": {
+    "BrowserCaptureMediaStreamTrack": {
+      "__compat": {
+        "spec_url": "https://w3c.github.io/mediacapture-region/#browser-capture-media-stream-track",
+        "support": {
+          "chrome": {
+            "version_added": "104"
+          },
+          "chrome_android": "mirror",
+          "edge": "mirror",
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": "mirror",
+          "ie": {
+            "version_added": false
+          },
+          "oculus": "mirror",
+          "opera": "mirror",
+          "opera_android": "mirror",
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": "mirror",
+          "samsunginternet_android": "mirror",
+          "webview_android": "mirror"
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "clone": {
+        "__compat": {
+          "spec_url": "https://w3c.github.io/mediacapture-region/#dom-browsercapturemediastreamtrack-clone",
+          "support": {
+            "chrome": {
+              "version_added": "104"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "cropTo": {
```

Comment:
This depends on `CropTarget`, added in https://github.com/mdn/browser-compat-data/pull/17678.